source/includes/steps-ui-create.yaml

```diff
@@ -1,35 +1,35 @@
-title: Navigate to the Realm Tab
+title: Navigate to the {+service-short+} Tab
 ref: realm-tab
 content: |
-  Go to the Project Dashboard where you want to create the Realm app. 
-  Select the :guilabel:`Realm` tab.
+  Go to the Project Dashboard where you want to create the {+app+}. 
+  Select the :guilabel:`{+service-short+}` tab.
```

Comment:
Will this tab in the UI actually say "App Services"? And we will also need to update screenshots that show "Realm" like the one further down this page, unfortunately. Presumably as part of a separate ticket, but should be a fast follow - maybe next week?

Reply:
It does say "App Services". We'll have to update screenshots.